lib/TAP/Formatter/Color.pm

```diff
@@ -12,7 +12,7 @@ my $NO_COLOR;
 BEGIN {
     $NO_COLOR = 0;
 
-    eval 'use Term::ANSIColor';
+    eval 'use Term::ANSIColor qw//';
```

Comment:
Changing code to make dzil happier sounds wrong, Prereqs::Soften sounds like a much better idea (I've previously used RemovePrereqs in such situations).